src/Symfony/Component/Messenger/Command/AbstractFailedMessagesCommand.php

```diff
@@ -122,6 +154,23 @@ protected function getReceiver(): ReceiverInterface
 
     protected function getLastRedeliveryStampWithException(Envelope $envelope): ?RedeliveryStamp
     {
+        if (null === \func_get_args()[1]) {
```

Comment:
this method only has one argument. isn't that accessing a non-existing argument which will result in a php warning?

Reply:
As discussed on Slack, I'll add a new PR in which I replace this check (that causes a warning if `\func_get_args()[1]` is not set) with a check on `\func_num_args`.